docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+{
```

Comment:
<blockquote>so we are only interested in the most-detailed entities in the cause and risk hierarchies.</blockquote><p>Well, we're <em>usually</em> interested in only the most-detailed entities, but there are always exceptions. For example, "Low Birthweight" and "Short Gestation" are the the most-detailed risks, but we work with "Low Birthweight and Short Gestation", one level up. So I would just say "we are usually only interested in the most detailed entities" or something like that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>